### ISSUE TITLE
Can't access administrative interface

### ISSUE THREAD

**Issue by ghost** [knowthelist/Growatt-server#2]
Hi there,

I hope someone here might know what is up.

I have a Growatt ShineWiFi-X and installed it on my inverter. I am able to put it into AP mode and see it's WiFi. However, there's a password on the WiFi. I did figure out the password (12345678), but after connecting to the WiFi, I am not able to access the administrative interface at 192.168.10.100. On my own device (laptop), I can see that I have the IP 192.168.10.101, so it seems I'm using the right IP. The browser just says: ERR_CONNECTION_REFUSED.

I've tried on a laptop and 2 different phones.

Any ideas?

**Comment by knowthelist** [comment]
to reach another client, you have to use the same sub net. This is not the case: 192.168.10.x vs. 192.168.1.x 
Or is that just a typo?

**Comment by ghost** [comment]
> to reach another client, you have to use the same sub net. This is not the case: 192.168.10.x vs. 192.168.1.x Or is that just a typo?

Yes, sorry. That was just a typo. When I connect to the AP, my device is handed the IP 192.168.10.101.
